CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+- TTID/TTFD Improvements ([#2866](https://github.com/getsentry/sentry-dart/pull/2866))
+```dart
+// To report TTFD you need to use the SentryNavigatorObserver.
+ 
+// Before reporting long running work, you can get a refernce to the currently displayed route.
+final sentryDisplay = Sentry.currentDisplay();
```

Comment:
```suggestion
// Before reporting long running work, you can get a reference to the currently displayed route.
final sentryDisplay = Sentry.currentDisplay();
```

pls explain that it has to be in a stateful widget within initState or if they use a stateless they should use the display widget instead

Reply:
does it even make sense to use this for a stateless widget?

Reply:
as discussed it makes sense to have stateless widgets assume it's TTFD is equal to TTID

Reply:
that means then we recommend using the SentryDisplayWidget generally